rankingOptions.ts

```diff
@@ -42,7 +42,7 @@ export const translations = {
     ],
     junior: [
         "Juniors",
-        "Junioren",
+        "Junior(inn)en",
```

Comment:
If i understand it correctly this translation is used for the profile of the pilot. Maybe it would be a nice way to just get the chosen gender and decide which term to use based on that

Reply:
I thought exactly about that. I think we will have problems with the terms especially where referring to singular instances (Pilot, Junior). Maybe it would be easiest if the frontend could choose between two terms.

Reply:
We should have a more consistent approach where we refer to multiple instances. Options would be Gender-Star (Junior*innen) or Binnen-I (JuniorInnen).

Reply:
Gender-Star is the way to go in my opinion.